Configuration/StandardSequences/python/Eras.py

```diff
@@ -55,7 +56,8 @@ def __init__(self):
                            'phase1Pixel', 'run3_GEM', 'run2_GEM_2017',
                            'run2_CSC_2018',
                            'phase2_common', 'phase2_tracker',
-                           'phase2_hgcal', 'phase2_muon', 'phase2_timing', 'phase2_hfnose', 'phase2_hgcalV10', 'phase2_hgcalV11', 'phase2_hgcalV12',
+                           'phase2_muon', 'phase2_GEM', 'phase2_GE0',
```

Comment:
Could you confirm you added `phase2_GEM` on purpose?
I don't see any other change related to `phase2_GEM`

Reply:
I would guess to clean up, we added phase2_GEM for the packer/unpacker earlier, but wasn't added here, so I guess Sven is taking the opportunity to add it now. @dildick this is from your commit, can you confirm?

Reply:
Hi, yes I added `phase2_GEM` here for completeness. We use it for the DQM and the unpacker.